### PR TITLE
POS: Add Delete Product btn and remove the icon

### PR DESCRIPTION
**Pull request by shubhamkmr04**
Issue: https://github.com/ZeusLN/zeus/issues/2199

Before:
Simulator Screenshot - iPhone 15 Pro - 2024-05-24 at 22 06 09

After:
Simulator Screenshot - iPhone 15 Pro - 2024-05-24 at 22 07 22
Simulator Screenshot - iPhone 15 Pro - 2024-05-24 at 22 07 26

### CODE REVIEW

**Comment by kaloudis** [comment]
@shubhamkmr04 can you provide before and after screenshots?

**Comment by shubhamkmr04** [comment]
> @shubhamkmr04 can you provide before and after screenshots?

Yes, updated the description!

File: views/POS/ProductDetails.tsx [hunk 1]
```diff
@@ -461,35 +431,40 @@ export default class ProductDetails extends React.Component<
...
+                                                this.state.confirmDelete
+                                                    ? localeString(
+                                                          'views.Settings.POS.confirmDelete'
+                                                      )
+                                                    : localeString(
+                                                          'views.Settings.POS.deleteProduct'
+                                                      )
+                                            }
+                                            onPress={() => {
+                                                if (!this.state.confirmDelete) {
```
Comment (kaloudis) [inline views/POS/ProductDetails.tsx:458]: nit: since we're referencing `this.state.confirmDelete` twice, let's destructure it on 227

**Review by kaloudis — CHANGES_REQUESTED** [review]

File: views/POS/ProductDetails.tsx (hunk 1 above)
Comment (shubhamkmr04) [reply views/POS/ProductDetails.tsx]: updated.

**Review by kaloudis — APPROVED** [review]

**Pull request merged**